persistence/src/main/scala/hmda/persistence/demo/DemoData.scala

```diff
@@ -85,6 +85,7 @@ object DemoData {
       case (instId: String, filings: Seq[Filing]) =>
         val filingActor = system.actorOf(FilingPersistence.props(instId))
         filings.foreach { filing => filingActor ? CreateFiling(filing) }
+        Thread.sleep(100)
```

Comment:
I'm curious about this.  Were you seeing test failures in the `InstitutionHttpApiSpec`?